README.md

```diff
@@ -277,6 +257,30 @@ the repository.
 | fixity.success|  It is also possible to trigger an action on success. By default, this is a no-op. The value should be a camel route action.  To log it to a file use something like this:  file:/tmp/?fileName=fixity-succes.log&fileExist=Append | null |
 | fixity.failure |  Most importantly, it is possible to configure what should happen when a fixity check fails. In the default example below, the fixity output is written to a file in `/tmp/fixityErrors.log`. But this can be changed to send a message to an email address (`fixity.failure=smtp:admin@example.org?subject=Fixity`) or use just about any other camel component.| file:/tmp/?fileName=fixity-errors.log&fileExist=Append |
 
+
+### Repository Audit Service (Triplestore)
+
+This application listens to Fedora's event stream, and stores
+audit-related events in an external triplestore. Both
+[Jena Fuseki](http://jena.apache.org/documentation/serving_data/)
+and [Open RDF Sesame](http://rdf4j.org/) are supported.
+
+More information about the
+[audit service](https://wiki.duraspace.org/display/FF/Design+-+Audit+Service)
+is available on the Fedora wiki.
+
+#### Properties
+| Name      | Description| Default Value |
+| :---      | :---| :----   |
+| audit.enabled | Enables/disables audit triplestore service  | false |
+| audit.input.stream | Audit Service jms message stream | broker:topic:fedora |
+| audit.event.baseUri | The baseUri to use for event URIs in the triplestore. A `UUID` will be appended to this value, forming, for instance: `http://example.com/event/{UUID}` | http://example.com/event |
+| audit.triplestore.baseUrl| The base url for the external triplestore service | http://localhost:3030/fuseki/test/update |
+| audit.triplestore.authUsername| Username for basic authentication against triplestore | |
+| audit.triplestore.authPassword| Password for basic authentication against triplestore | |
+| audit.filter.containers |  A comma-delimited list of URIs to be filtered (ignored) by the audit service | http://localhost:8080/fcrepo/rest/audit | 
+
+
 ## Building
```

Comment:
s/Building/Troubleshooting

This was probably from an incorrect conflict resolution before.